### PR TITLE
Fixed all tags in repository

### PR DESCRIPTION
**Pull request by edsonarios**
Issue link: https://github.com/softlayer/githubio_source/issues/532

Created script and executed to fix all tags in all files for change to lowercase

Observation: 
The script is already executed, and it is not necessary to execute it again for now, but it was added to the repository, if it is ever needed again

### CODE REVIEW

File: content/python/networkMapper.md [hunk 1]
```diff
@@ -13,69 +13,69 @@ tags:
...
-        dcs = self.client['SoftLayer_Location_Datacenter'].getDatacenters()
+        dc_mask="mask[backendhardwarerouters]"
+        dcs = self.client['softlayer_location_datacenter'].getdatacenters()
 
         for dc in dcs:
             print("%s" % (dc['name']))
-            router_mask = "mask[hardwareChassis,downlinkHardware[hardwareChassis,downlinkHardware[hardwareChassis,downstreamServers[id,fullyQualifiedDomainName],id,fullyQualifiedDomainName]]]"
-            backend = self.client['SoftLayer_Location_Datacenter'].getBackendHardwareRouters(id=dc['id'],mask=router_mask)
+            router_mask = "mask[hardwarechassis,downlinkhardware[hardwarechassis,downlinkhardware[hardwarechassis,downstreamservers[id,fullyqualifieddomainname],id,fullyqualifieddomainname]]]"
+            backend = self.client['softlayer_location_datacenter'].getbackendhardwarerouters(id=dc['id'],mask=router_mask)
```
Comment (BrianSantivanez) [inline content/python/networkMapper.md:59]: Is it correct to change "getBackendHardwareRouters" method by "getbackendhardwarerouters" method?

File: scriptsTools/convertAllTagsToLowercase.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
+let shell = require('shelljs')
+const fs = require('fs')
+
+const ROUTE_FILES = 'C:/Users/LVV941629/Forks/Edson/githubio_source/content'
```
Comment (BrianSantivanez) [inline scriptsTools/convertAllTagsToLowercase.js:4]: Will ROUTE_FILES work on all devices when the project runs?

File: content/python/networkMapper.md (hunk 1 above)
Comment (edsonarios) [reply content/python/networkMapper.md:59]: You are right, this file should not have changed, only the tags

File: scriptsTools/convertAllTagsToLowercase.js (hunk 2 above)
Comment (edsonarios) [reply scriptsTools/convertAllTagsToLowercase.js:4]: Good observation, each one must place the path of their project, that will be specified

**Review by BrianSantivanez — APPROVED** [review]
Looks good to me

**Review by allmightyspiff — APPROVED** [review]
Looks good, thanks

**Pull request merged**